### PR TITLE
Fix: change the errored action name and add the e2e test

### PR DESCRIPTION
**Pull request by barnettZQG**
Signed-off-by: barnettZQG <barnett.zqg@gmail.com>

### Description of your changes

Fixes #

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Review by wonderflow — APPROVED** [review]

**Review by wonderflow — COMMENTED** [review]
no tests?

File: pkg/stdlib/pkgs/config.cue [hunk 1]
```diff
@@ -19,7 +19,7 @@
 }
 
 #Read: {
-	#do:       "delete"
```
Comment (FogDong) [inline pkg/stdlib/pkgs/config.cue:22]: Please add some suite test.

File: e2e/config_test.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+			time.Second*30, time.Second*2).Should(Equal(v1alpha1.WorkflowStateSucceeded))
+	})
+
+	AfterEach(func() {
+		By("Clean up resources after a test")
+		k8sClient.DeleteAllOf(ctx, &v1alpha1.WorkflowRun{}, client.InNamespace(namespace))
+	})
+})
+
+var manageConfigWorkflowRun = `
```
Comment (FogDong) [inline e2e/config_test.go:70]: Can we move this to `/testdata`?

File: e2e/e2e_suite_test.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+	k8sClient, err = client.New(conf, client.Options{Scheme: scheme})
+	Expect(err).Should(BeNil())
+
+	prepareWorkflowDefinitions()
+})
+
+func prepareWorkflowDefinitions() {
+	var files []string
+	err := filepath.Walk("./definitions/", func(path string, info os.FileInfo, err error) error {
```
Comment (FogDong) [inline e2e/e2e_suite_test.go:56]: You can use help function from `kubevela/pkg` like: https://github.com/kubevela/workflow/blob/main/controllers/workflow_test.go#L1545

Comment (barnettZQG) [reply e2e/e2e_suite_test.go]: Great!

**Pull request merged**